src/_P020_Ser2Net.ino

```diff
@@ -54,7 +54,7 @@ boolean Plugin_020(uint8_t function, struct EventStruct *event, String& string)
       Device[deviceCount].InverseLogicOption = false;
       Device[deviceCount].FormulaOption      = false;
       Device[deviceCount].ValueCount         = 0;
-      Device[deviceCount].SendDataOption     = true;
+      Device[deviceCount].SendDataOption     = false;
```

Comment:
Why would you change this flag? Doesn't seem backward compatible to me.

Reply:
clean up. The user had an otion which did nothing.

Reply:
I do not agree.
This is to allow sending data to a controller.
Now it is no longer possible to send data to a controller

Reply:
ok, I do not understand to which controller could make use of ser2net; nevertheless, I set it back

Reply:
done